lib/jazzy/source_declaration/type.rb

```diff
@@ -5,7 +5,7 @@ class SourceDeclaration
     # rubocop:disable Metrics/ClassLength
     class Type
       def self.all
-        TYPES.keys.map { |k| new(k) }
+        TYPES.keys.reject { |k| k == 'Overview' }.map { |k| new(k) }
```

Comment:
How about doing the map as before, then rejecting `Type`s with nil name?  Reduces number of times 'Overview' is hard-coded + bit more future proof.

Reply:
Sounds good